### PR TITLE
Delete rule `ack` endpoint

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Delete rule `ack` endpoint - simpler than previous one :)

## Type of change

- New feature (non-breaking change which adds functionality)

## Testing steps

N/A

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: server/api/v2/openapi.json [hunk 1]
```diff
@@ -287,6 +287,36 @@
         }
       }
     },
+    "/ack/{rule_id}": {
+      "delele": {
```
Comment (joselsegura) [inline server/api/v2/openapi.json:291]: s/delele/delete/

**Review by joselsegura — CHANGES_REQUESTED** [review]

**Review by joselsegura — COMMENTED** [review]
Only that nitpick on the openapi.json.

LGTM overall

File: server/api/v2/openapi.json (hunk 1 above)
Comment (tisnik) [reply server/api/v2/openapi.json:291]: aaah!

Comment (tisnik) [reply server/api/v2/openapi.json]: FixeD!

**Review by joselsegura — APPROVED** [review]

File: server/acks_handlers.go [hunk 2]
```diff
@@ -133,3 +133,56 @@ func (server *HTTPServer) acknowledgePost(writer http.ResponseWriter, request *h
...
 	returnRuleAckToClient(writer, updatedAcknowledgement)
 }
+
+// method deleteAcknowledge deletes an acknowledgement for a rule, by its rule
+// ID. If the ack existed, it is deleted and a 204 is returned. Otherwise, a
+// 404 is returned.
+func (server *HTTPServer) deleteAcknowledge(writer http.ResponseWriter, request *http.Request) {
+	orgID, userID, err := server.readOrgIDAndUserIDFromToken(writer, request)
+	if err != nil {
+		// everything's handled already
```
Comment (dpensi) [inline server/acks_handlers.go:143]: not even a log ?

**Review by dpensi — APPROVED** [review]

File: server/acks_handlers.go (hunk 2 above)
Comment (tisnik) [reply server/acks_handlers.go:143]: I'll add, yes

**Review by dpensi — APPROVED** [review]

**Pull request merged**